api/src/main/java/org/openmrs/Encounter.java

```diff
@@ -384,6 +386,24 @@ public Patient getPatient() {
 	public void setPatient(Patient patient) {
 		this.patient = patient;
 	}
+
+	/**
+	 * Gets the set of diagnoses
+	 * @return diagnoses - the set of diagnoses.
+	 */
+	
+	public Set<Diagnosis> getDiagnoses() {
+		return diagnoses;
+	}
+
+	/**
+	 * Sets a set of diagnoses for the current Encounter
+	 * @param diagnoses the set of Diagnosis to set.
+	 */
+	
```

Comment:
The javadoc block should be right next to the method. That is no empty line between.